python/tests/test_options.py

```diff
@@ -19,6 +19,13 @@ def test_accepts_browser_version_platform_name(self):
         assert sauce.browserVersion == 'latest'
         assert sauce.platformName == 'Windows 10'
 
+    def test_accepts_browser_version_platform_name(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'test_accepts_browser_version_platform_name' from line 15](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)